### ISSUE TITLE
Option to disable Animated Icons - Fire 7 Tablet (2019) Issue

### ISSUE THREAD

**Issue by sstratoti** [bramkragten/weather-card#29]
I'm running some HA Lovelace Dashboards using an Amazon Fire Tablet. I noticed that the tablet that I was using kept flickering, every once and a while.
I noticed on FullyKiosk's website (the app I'm using for a kiosk dashboard) that it states:

> Android 4.4 contains an outdated and buggy Chromium web engine. There are some display problems known, eg with SVG fonts. Unfortunately it’s not fixable because the Android Webview can’t be updated in Android 4.4.

I cloned your repository, and uploaded the static icons you provided to a custom ui folder. Then I referenced them in your readme like you suggested. Boom - no more flickering. I think it has something to do with animated SVGs in general.

The SVGs look great, and I really love the layout of the weather card on the tablet. It'd be great if others could turn it off via a setting rather than having to implement the workaround that I had to. 

Thanks!

**Comment by david-kalbermatten** [comment]
Yea, having a simple toggle in the configuration screen would be highly appreciated 👍 (Especially since the animated icons seem to cause an extreme CPU load)

**Comment by bramkragten** [comment]
You can choose a different icon location and point it to the none animated icons.

**Comment by bramkragten** [comment]
`icons: https://cdn.jsdelivr.net/gh/bramkragten/weather-card@latest/icons/static/`

**Comment by sstratoti** [comment]
Yup, that's what I am currently doing. I was suggesting that it be a setting rather than a workaround. 

`animated: true 
`

**Comment by AlesZurek** [comment]
I just wanted to say that I have same issue with my Nexus 7 (2013) tablet mounted on wall. Once I have the weather-card with animated icons and it is running for a while I got Android error that Home assistant application is not responding with options to Wait or close it when I turn on the screen.
Changing the icons location (that post from @bramkragten above) help me to solve it.
